l10n_ch_account_statement_base_import/parsers/ubs_file_parser.py

```diff
@@ -118,8 +118,8 @@ def _parse_transactions(self):
                 line.get("Description 3", '')]
             label = ' '.join(filter(None, descriptions))
 
-            debit = "D�bit".decode('iso-8859-15')
-            credit = "Cr�dit".decode('iso-8859-15')
+            debit = "Débit".decode('iso-8859-15')
+            credit = "Crédit".decode('iso-8859-15')

```

Comment:
Indeed it must be some encoding change. As we see no difference

file old_file.py 
old_file.py: Python script, ISO-8859 text executable, with CRLF line terminators

file new_file.py 
new_file.py: Python script, UTF-8 Unicode text executable, with CRLF line terminators

So the new file is correct.

You might just want to put:

 u"Débit".encode('iso-8859-15')